### PR TITLE
sys-boot/raspberrypi-firmware: simple bump to 1.20201201

### PR DESCRIPTION
**Pull request by crabbedhaloablution**
Closes: https://bugs.gentoo.org/759262
Package-Manager: Portage-3.0.12, Repoman-3.0.2
Signed-off-by: Peter Alfredsen <crabbedhaloablution@icloud.com>

### CODE REVIEW

**Comment by expeditioneer** [comment]
Should this ebuild not depend on sys-kernel/raspberrypi-sources--${PV} or sys-kernel/raspberrypi-image-${PV}?

**Comment by crabbedhaloablution** [comment]
> Should this ebuild not depend on sys-kernel/raspberrypi-sources-${PV} or sys-kernel/raspberrypi-image-${PV}?

No. The files contained in this package are better thought of as the bootloader to the kernel. They will load any kernel that lives up to their specifications.

File: sys-boot/raspberrypi-firmware/raspberrypi-firmware-1.20201201.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+			msg+="/boot/cmdline.txt "
+		fi
+
+		if [[ -e "${ED}"/boot/config.txt ]] && [[ -e /boot/config.txt ]] ; then
+			msg+="/boot/config.txt "
+		fi
+
+		if [[ -n "${msg}" ]] ; then
+			msg="This package installs following files: ${msg}."
+			msg="${msg} Please remove (backup) your copies durning install"
```
Comment (expeditioneer) [inline sys-boot/raspberrypi-firmware/raspberrypi-firmware-1.20201201.ebuild:62]: typo: 'durning'

Comment (crabbedhaloablution) [reply sys-boot/raspberrypi-firmware/raspberrypi-firmware-1.20201201.ebuild]: Fixed

**Pull request closed without merging**